opennms-config-model/src/main/java/org/opennms/netmgt/config/eventd/EventdConfiguration.java

```diff
@@ -127,6 +127,36 @@
 
     @XmlValue
     private String m_contents;
+    
+    
+    /**
+     * Number of threads used for consuming/dispatching messages.
+     * Defaults to 2 x the number of available processors.
+     */
+    @XmlAttribute(name = "threads", required=false)
```

Comment:
Can we prefix these new attribute with `sink-` to make it clear that these settings are used for the Sink API consumer?

Reply:
we followed the legacy of how it is handled and described in syslogd-configuration.xml and trapd-configuration.xml for example https://github.com/OpenNMS/opennms/blob/develop/opennms-base-assembly/src/main/filtered/etc/trapd-configuration.xml

Reply:
In those cases all of the traps/syslogs are sourced from the Sink API, so there's no ambiguity as to what threads we're configuring. In the case of eventd there are a number of existing threads pools, and various sources of events. Prefixing these attributes will help make whats being configured clearer.